0/setup.md

```diff
@@ -16,6 +16,9 @@ curl https://getsubstrate.io -sSf | bash -s -- --fast
 ```
 rustup target add wasm32-unknown-unknown --toolchain stable
 rustup component add rust-src --toolchain nightly
+rustup toolchain install nightly-2020-06-01
+rustup target add wasm32-unknown-unknown --toolchain nightly-2020-06-01
```

Comment:
Why are we using such an old nightly version?
We should always propagate the newest available nightly that works for us which currently should be `2020-08-26`.

Reply:
It is only used to build the Substrate `rc4` binary. For the contracts it will stay with the newer nightly.
Issues of supporting old Substrate releases, after some time things start breaking.